pvr.mediaportal.tvserver/resources/settings.xml

```diff
@@ -16,7 +16,9 @@
     <setting id="tvgroup" type="text" label="30006" default="" />
     <setting id="radiogroup" type="text" label="30007" default="" />
     <setting id="readgenre" type="bool" label="30009" default="false" />
-	<setting id="enableoldseriesdlg" type="bool" label="30011" default="false" />
+	  <setting id="enableoldseriesdlg" type="bool" label="30011" default="false" />
```

Comment:
Whitespace issue

Reply:
This issue was commited before :-) i will fix it